### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by Ilyza91** [emmeken/abyss-diver#1]
Hey! I'm testing the setup! I feel like an idiot but the batch program seems to work, but I don't end up with a html file anywhere. Shouldn't it just end up in the cloned local repo file?

**Comment by emmeken** [comment]
The "build.sh" script should print "Compiling to: Abyss Diver.html"; the output file being "Abyss Diver.html".

Are you running the script from a terminal or from a file manager? Please run it from a terminal, so you can see any output or errors that it prints.

**Comment by Ilyza91** [comment]
So I tried to push an update, but I guess I need writing permission from you first. I created a fork for now, just to test a few things.

**Comment by emmeken** [comment]
A fork is fine: I don't think mine should be the main repo anyway in the long run; I only created it to show what is possible and to provide a starting point.

Did you get the compilation to work?

**Comment by Ilyza91** [comment]
Haha no not yet. I don't get any errors, I really don't get it. It did work for you? I'll give it another go on monday when I have some time

**Comment by emmeken** [comment]
Yes, it works here: it produces an HTML file and that file plays the game just fine.

You are running it from a terminal / command prompt, right? Because otherwise you won't see any error messages.

**Comment by Ilyza91** [comment]
It was a version issue in the end, which I just overlooked in the output. But it works now thanks!

**Comment by Ilyza91** [comment]
If you can update the repo again with 8.5 I'll ask floric to also come over, so we can start working from here. Thanks a lot!

**Comment by emmeken** [comment]
I sent a [pull request](https://github.com/Ilyza91/abyss-diver/pull/1) which contains the 0.8.51 changes. If you accept that, those changes will be merged into your repo.
